andrezero/boilerplate-esm-nyc-mocha
# Upgrade dependencies

This PR upgrades the participating dependencies to latest

#### Background

I was trying to figure out why `nyc` failed to report coverage properly in my repository, when I came across this repository.

Thinking that it might recent changes that caused `nyc` to miscalculate for my repository, I tried upgrading dependencies in this repository. It still works just fine

#### Solution

Upgrade dependencies, so others might not spend as long on it as I have

#### How to verify

1. Check out this branch
1. `npm ci`
1. `npm test`
1. Observe that `nyc` still calculates coverage correctly for all files
